artifact_env.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# This script creates a conda environment from a requirements.txt file from Github.
+# Conda environments should be created with the simulation science service user and 
+# the credientials can be found at /mnt/team/simulation_science/priv/credentials/svc-simsci
+
+# Three arguments must be provided to the script:
+#   - The first argument is the name of the conda environment
+#   - The second argument is the repo name
+#   - The third argument is the name of the branch to fetch requirements.txt from for
```

Comment:
I'm not sure I completely understand what you are saying but the idea is: We will have a requirements.txt and you can edit it to target specific branches in both the model repo and the framework repos. 

So for example lets say you want to test out a new feature in VPH and the implementation for that in the model repo both of which have a branch new-feature-branch. In requirements.txt you can specify those branches for VPH and the model repo and then build the environment.

Currently, we should expect to target main for the model repo and release-candidate for vivarium, VPH, and VCT (although at the moment the model repo is not targetting main because requirements.txt doesn't exist in main).

Reply:
But even as part of this PR you have two differently-named requirements text files so this would never load the artifact one, right?

Reply:
The artifacts.txt is linked in the artifact env shell script where the simulation environment creation shell script links to the normal requirements.txt

Reply:
So ou could build from different branches from the artifact vs simulation environments